mmv1/products/filestore/Instance.yaml

```diff
@@ -213,6 +213,10 @@ properties:
             A /29 CIDR block that identifies the range of IP
             addresses reserved for this instance.
           default_from_api: true
+          # When using connectMode=PRIVATE_SERVICE_ACCESS, the returned value
+          # won't match the sent value as the returned value is an IP range
+          # from the provided named range.
+          ignore_read: true
```

Comment:
Looks like `ignore_read` does not work for nested fields currently based on https://github.com/hashicorp/terraform-provider-google/issues/12410#issue-1352808232. As it's suggested in the issue, you'd probably want to handle it with `custom_flatten` similar to https://github.com/GoogleCloudPlatform/magic-modules/blob/main/mmv1/products/compute/BackendService.yaml#L756 to write back the value set in config, which should resolve the permadiff issue.

Reply:
Thanks @shuyama1 for pointing this our and adding the fix.